argocd/overlays/moc-infra/projects/operate-first.yaml

```diff
@@ -75,3 +75,7 @@ spec:
       kind: ThanosRuler
     - group: logging.openshift.io
       kind: ClusterLogForwarder
+    - group: operators.coreos.com
+      kind: OperatorGroup
+    - group: operators.coreos.com
+      kind: Susbscription
```

Comment:
the subscription and operatorgroup are not being deployed by the app you are creating here, but rather the `cluster-resources-smaug`, which already has the permissions to deploy these, so you can revert the changes to this project.

Reply:
Thanks! Should be fixed now.